include/SFML/System/String.hpp

```diff
@@ -56,7 +56,8 @@ class SFML_SYSTEM_API String
     ////////////////////////////////////////////////////////////
     // Static member data
     ////////////////////////////////////////////////////////////
-    static const std::size_t InvalidPos; //!< Represents an invalid position in the string
+    static inline const std::size_t InvalidPos{
+        std::basic_string<std::uint32_t>::npos}; //!< Represents an invalid position in the string
```

Comment:
Is this driven by clang formatting? Looks rather unreadable to me.

Reply:
Yeah that's clang-format failing to fit the expression and its trailing comment within our width limit. For the record, removing `std::` from `size_t` and `uint32_t` allow clang-format to fit this whole expression on one line. 

I moved that comment up one line and tweaked it such that it still produces the correct documentation. This is what the docs look like after this change.

<img width="542" alt="Screen Shot 2022-09-29 at 2 59 44 PM" src="https://user-images.githubusercontent.com/39244355/193140966-0dc643e9-dc54-4240-8599-3780c1d4bcf0.png">

Reply:
You dropped one `std::` in the process.
You can have `static inline const std::size_t InvalidPos{std::basic_string<std::uint32_t>::npos};` on one line.

Reply:
Sorry! It's fixed now.

Reply:
Also, starting documentation comment with `///` would me more consistent with the rest of the code base.

Reply:
Are there any other member variables we document on the preceding line? I wasn't aware of any so I just went with what worked first.

Reply:
As far as I know, the convention is to use `///` except when putting documentation after member because some IDE would parse `///<` as XML, so we use `//!<` there.

Reply:
Alright I can do that. I tried it locally and Doxygen still produces the same output.